### PR TITLE
Add support for DynamoDB tables

### PR DESCRIPTION
**Pull request by reedloden**

### CODE REVIEW

File: internal/awsdata/data.go [hunk 1]
```diff
@@ -263,6 +270,19 @@ func AppendIfMissing(slice []string, s string) []string {
 	return append(slice, s)
 }
 
+func HumanReadableBytes(b int64) string {
```
Comment (reedloden) [inline internal/awsdata/data.go:288]: Another random util function to be moved to a separate file at some future point.

File: internal/awsdata/dynamodb.go [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+			return
+		}
+
+		d.results <- result{
+			Row: inventory.Row{
+				UniqueAssetIdentifier:  aws.StringValue(out.Table.TableName),
+				Virtual:                true,
+				Location:               region,
+				AssetType:              AssetTypeDynamoDBTable,
+				HardwareMakeModel:      HumanReadableBytes(aws.Int64Value(out.Table.TableSizeBytes)),
```
Comment (itmecho) [inline internal/awsdata/dynamodb.go]: Would this be better as `Row.Comments`?

**Review by itmecho — CHANGES_REQUESTED** [review]
Looks good to me! One thing to note with these is that the tests don't cover paginated responses. I'm happy to merge it for now and fix this afterwards though

File: internal/awsdata/data.go [hunk 3]
```diff
@@ -116,6 +118,11 @@ func (d *AWSData) Load(regions, services []string) {
 
 	// Regional Services
 	for _, region := range regions {
+		if stringInSlice(ServiceDynamoDB, services) {
+			d.wg.Add(1)
```
Comment (itmecho) [inline internal/awsdata/data.go:124]: I've just merged some logging improvements to make reviewing PRs a easier, would you mind adding the new log line here?

https://github.com/manywho/awsinventory/blob/339a13886b7b50a894306138493eb3b54ca9466b/internal/awsdata/data.go#L121

File: internal/awsdata/dynamodb.go (hunk 2 above)
Comment (reedloden) [reply internal/awsdata/dynamodb.go]: EBS rows include the size as part of `HardwareMakeModel`, which is why I included it here, but I have no issue with putting it under `Comments`. No preference either way.

**Comment by reedloden** [comment]
> Looks good to me! One thing to note with these is that the tests don't cover paginated responses. I'm happy to merge it for now and fix this afterwards though

Yeah, definitely been on my mind from my last PR. I welcome thoughts on the best way to do this, as I have a few potential ideas. Maybe I'll look at how Terraform does it.

File: internal/awsdata/dynamodb.go (hunk 2 above)
Comment (itmecho) [reply internal/awsdata/dynamodb.go]: Hmm thinking about it, I guess it's hardware information and we don't really have any better data to put here so should be fine to leave it. We can always change it in the future if it gets flagged in any FedRAMP audits

Comment (itmecho) [reply internal/awsdata/dynamodb.go]: Oh you already changed it=p Nevermind, it can stay in Comments

**Comment by itmecho** [comment]
I've got an idea, I'll create a test PR and ask your opinion. Thanks for this one!

**Review by itmecho — APPROVED** [review]

**Pull request merged**